src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -164,4 +164,8 @@
     </StringProperty>
 
     <DynamicEnumProperty Name="SupportedTargetFrameworks" DisplayName="Supported TargetFrameworks" EnumProvider="SupportedTargetFrameworksEnumProvider" Visible="False"/>
+    
+    <!-- F# specific properties-->
+    <BoolProperty Name="CanUseTargetFSharpCoreVersion" Visible="False"/>
```

Comment:
Just because I don't know how the rules work - where does this get made visible for F#?